Python/Certificate_Generator/Readme.MD

```diff
@@ -0,0 +1,44 @@
+# Certificate Gerneration Script
+
+## Introduction
+
+Generating a ton of certificates for any occassion is quite tedious if you're not familiar with the concept of automation. Well, you won't have to worry anymore because this script will do all the heavy lifting for you.
+
+
+## How to get started?
+
+In order to run this script, you need to have Python 3.7.x installed in your machine. You can get it from [here](https://www.python.org/downloads/). After you're done installing Python 3.7.x, run the following command from your terminal to install [OpenCV](https://pypi.org/project/opencv-python/):
+
+```
+pip install opencv-python
+```
+```
+Now in the namelist.txt file just add the name for whom you want to generate the certificate
+if you want different template add a template file as template.png in the same directory
+```
+
+## Great, you're almost there!
+
+
+## Run the script
+
+Open your terminal and navigate to the directory where you downloaded this project. After that, run the following command from your terminal to generate your certificates:
+
+```

```

Comment:
specify to the user that they need to add a template certificate and rename it accordingly.